YoCode/GitCheck.cs

```diff
@@ -8,18 +8,14 @@ namespace YoCode
 {
     public class GitCheck
     {
-        public string GIT_INSTALL_DIRECTORY { get; set; } = @"C:\Program Files\Git";
-        //public string REPOSITORY_PATH = @"..\..\..\..\..\YoCode";
         public string REPOSITORY_PATH = @"C:\Users\ukmzil\source\repos\Tests Sent by People\Real\simon-jones";
```

Comment:
Is this repository path going to be fixed after merging? It should come from the "modified path" that the user specified when they ran the program.

Reply:
so far its hardcoded, until the merge i suppose